app/views/layouts/_header.mobile.haml

```diff
@@ -4,32 +4,38 @@
       = link_to(image_tag("mobile/asterisk_white_mobile.png", class: "img-responsive"),
           stream_path, id: "header-title", class: "navbar-brand")
 
-      %ul.nav.navbar-nav#nav-badges
-        -# Notifications
-        %li
-          = link_to notifications_path, class: "badge-link", id: "notification-badge" do
-            %i.entypo-bell
-            - if current_user.unread_notifications.size > 0
-              %span.badge.badge-important#notification
-                = current_user.unread_notifications.size
+      - if current_user
```

Comment:
This test is the one used on desktop, but application.haml.mobile uses `user_signed_in?`, which one should I use?